src/actions/commands/insert.ts

```diff
@@ -530,3 +530,29 @@ class CreateUndoPoint extends BaseCommand {
     vimState.historyTracker.finishCurrentStep();
   }
 }
+
+@RegisterAction
+export class CommandMoveCursorOneLineDownInInsertMode extends BaseCommand {
+  modes = [Mode.Insert];
+  keys = [
+    ['<C-g>', 'j'],
+    ['<C-g>', '<down>'],
+  ];
+
+  public override async exec(position: Position, vimState: VimState): Promise<void> {
+    vimState.cursorStopPosition = position.getDown(1);
```

Comment:
This doesn't seem to match Vim's behavior, which is to go one line up (or down), at the column Insert mode began in